### ISSUE TITLE
'geos/platform.h' file not found

### ISSUE THREAD

**Issue by fbirot** [andreacremaschi/SpatialDBKit#24]
Hi there,

I just downloaded SpatialDBKit, installed cocoapods and made sure I had automake autoconf and libtool installed. When I do a "pod install" in the "SpatialDBKitExample" directory, it downloads and installs dependencies. Yet when I try to compile the SpatialDBKitExample project, I get the following error:

"Coordinate.h:19:10: 'geos/platform.h' file not found"

When I look in the "SpatialDBKitExample/Pods/geos/include/geos" folder, I indeed don't have any platform.h file, but only a platform.h.in file (same problem with version.h that is missing but has its version.h.in file).

I'm extremely new to cocoapods and don't know what it does when it "installs" geos (I can't see any configure.in file), but it seems that autoconf doesn't generate the expected platform.h file (nor the version.h one).

Any idea of what is happening ?

**Comment by andreacremaschi** [comment]
mh I just tried and everything looks.. well.. "ok" :)
these issues are still open: https://github.com/andreacremaschi/SpatialDBKit/issues/15
and you should rename main.m in main.mm
but not the one you point out. Which version of cocoa pods are you using?

**Comment by andreacremaschi** [comment]
Everything looks ok for me.. You have to change spatialite.c:87 to 

```
#include <spatialite/spatialite/spatialite.h>
```

but this fix an error that is not the one you describe, so I suppose you already did it. Which version of cocoa pods are you using?

**Comment by fbirot** [comment]
Thank you for your answer. 

I re-cloned the repository, tried again and it worked (sorry for not having tried that before).

I actually had an error the first time I executed "pod install", because it couldn't find autoconf. After fixing that I thought I only needed to execute "pod install" again, and that it should work like it was the first time (since I couldn't find any "pod uninstall" command), but apparently not. It apparently kept some bad configuration somewhere. I had even tried to delete Pods/geos before executing "pod install", but still had this problem. 

But anyway, it now works :) Thank you for your time!
